pkg/controller/appdefinition/deploy.go

```diff
@@ -406,9 +407,32 @@ func toContainer(app *v1.AppInstance, tag name.Reference, deploymentName, contai
 		LivenessProbe:  toProbe(container, v1.LivenessProbeType),
 		StartupProbe:   toProbe(container, v1.StartupProbeType),
 		ReadinessProbe: toProbe(container, v1.ReadinessProbeType),
+		Resources:      toResources(req, app, container, containerName),
 	}
 }
 
+func toResources(req router.Request, app *v1.AppInstance, container v1.Container, containerName string) corev1.ResourceRequirements {
+	var bytes int64
```

Comment:
nit: `bytes` here may be confusing because of the `byte` type in Go. Suggest using `memBytes` instead.